### PR TITLE
WZDx v3.1 — Spring 2021 Release

### PR DESCRIPTION
**Pull request by j-d-b**
This PR is for the v3.1 specification release.

All specific PRs for the changes for this release have been merged into the working branch `spring-2021-release` and reviewed over the last two months. With the close of working group voting this week, all changes implemented in `spring-2021-release` have been approved and now the release branch can be merged into master to mark the official release.

Details of the changes for this release are below, taken from `RELEASES.md`.

---

Following the major v3.0 release, the changes for the v3.1 release were developed with the goal enhancing and cleaning up the specification without making major changes or breaking backwards compatibility. Thus, in v3.1 many enumerated type values or object properties were deprecated rather than removed. Most if not all of these deprecated values will be removed in the next major release.

This repository was also renamed from `jpo-wzdx` to `wzdx` on 2021-04-05. All links pointing to `jpo-wzdx` will automatically redirect to the new URL. 

## Features
* Add `local-access-only` restriction
* Add `license` property to the `RoadEventFeedInfo` object

## Refactoring
* Refactor `LaneType` enumerated type to deprecate values that can be determined from other properties of the Lane object, such as `order`, `status`, and `lane_restrictions`
* Add `road_names` property to the `RoadEvent` object and deprecate `road_name` and `road_number`
* Deprecate the `total_num_lanes` property on the `RoadEvent` object as the `RoadEvent`'s `lanes` array can be used to determine the number of lanes

## Fixes
* Add optional `bbox` property to allow providing a [GeoJSON Bounding Box](https://tools.ietf.org/html/rfc7946#section-5) for the `WZDxFeed` and `RoadEventFeature` objects
* Add an `id` property to the `RoadEventFeature` object for providing the a road event's identifier to better follow [GeoJSON ID recommendations](https://tools.ietf.org/html/rfc7946#section-3.2)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,54 +92,56 @@ This project repository will be continually updated with resources to help with
...
 
-- [WZDI Framework](https://collaboration.fhwa.dot.gov/wzmp/Framework/Forms/AllItems.aspx) provides a conceptual architecture for work zone data systems for collecting, storing, disseminating, managing, maintaining and archiving work zone activity data.
-- [WZDI Data Dictionary](https://collaboration.fhwa.dot.gov/wzmp/Data%20DictionaryDocuments/Forms/AllItems.aspx) provides digital descriptions of work zone activities that enable and support transportation agencies and third party providers to describe and communicate work zone-related information to agency, private sector, and public users timely and seamlessly across multiple jurisdictions and regions.
+- WZDI Framework provides a conceptual architecture for work zone data systems for collecting, storing, disseminating, managing, maintaining and archiving work zone activity data.
```
Comment (j-d-b) [inline README.md:95]: @Mahsa-Ettefagh did you intentionally remove these links?

Comment (natedeshmukhtowery) [reply README.md:95]: These links are broken, currently, as the FHWA collaboration site in question is shut down. We are working to make these documents available directly here; @wesleyalforddot is working with FHWA to obtain the current versions. Thanks

Comment (j-d-b) [reply README.md:95]: thanks @natedeshmukhtowery

File: create-feed/schemas/wzdx_v3.1_feed.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 {
-  "$id": "https://github.com/usdot-jpo-ode/jpo-wzdx/tree/spring-2021-release/create-feed/schemas/wzdx_vNext_feed.json",
+  "$id": "https://raw.githubusercontent.com/usdot-jpo-ode/wzdx/master/create-feed/schemas/wzdx_v3.1_feed.json",
```
Comment (j-d-b) [inline create-feed/schemas/wzdx_v3.1_feed.json:3]: @chuehlien great change

**Review by chuehlien — APPROVED** [review]
Good work @j-d-b !

I made a couple of changes - please review the languages in the following commits and feel free to edit as you see fit. Thanks!
- additions to release note re: repo renaming (https://github.com/usdot-jpo-ode/wzdx/pull/168/commits/07a5b0efdca736fb5ce541ca3bae0c3d4f8459ed) and LaneStatus (https://github.com/usdot-jpo-ode/wzdx/pull/168/commits/e7e87466c524349fc49880c73a3c9a653868d763)
- modified README paragraph re: feed URLs (https://github.com/usdot-jpo-ode/wzdx/pull/168/commits/ec262c50683c9385fdaa894fd793cce33d07b361)

**Pull request merged**